pkg/proxy/nftables/proxier.go

```diff
@@ -975,6 +998,89 @@ func isAffinitySetName(set string) bool {
 	return strings.HasPrefix(set, servicePortEndpointAffinityNamePrefix)
 }
 
+type containerArray interface {
+	[0]string | [1]string | [2]string | [3]string
+}
+
+// nftContainer is an internal representation of map or set.
```

Comment:
"container" in pretty much any kubernetes context implies docker/OCI containers, so this naming is confusing.